src/ansys/aedt/core/aedt_logger.py

```diff
@@ -801,6 +802,25 @@ def add_logger(self, destination, level=logging.DEBUG):
         else:
             raise ValueError("The destination must be either 'Project' or 'Design'.")
 
+    @property
+    def log_on_desktop(self):
+        """Status of the log in AEDT (Message Manager).
+
+        Returns
+        -------
+        bool
+            True if logging to AEDT is enabled, False otherwise.
```

Comment:
Small details to be consistent across docstrings. Otherwise, it is great.
```suggestion
            ``True`` if logging to AEDT is enabled, ``False`` otherwise.
```